digitalocean-app-platform/README.md

```diff
@@ -0,0 +1,124 @@
+
+#### If you prefer to use DigitalOcean's Managed redis Database solution:
+
+* Under ```Manage``` in the left-hand navigation menu, select ```Databases``` or select the ```Create``` dropdown menu in the top right corner of your DigitalOcean tenant and select ```Databases```.
+* Choose redis as your Database Engine.
+* Under ```Choose your Configuration```, leaving the ```Machine Type``` set to the ```Basic Nodes``` option is sufficient.
+* Choose a Datacenter.
+* Once you've configured the other settings on this page to your liking, click ```Create a Database Cluster```.
+
+Once the creation process of your managed database is complete:
+
+* Click on the hostname of your new container from your list of managed databases.
+* In the top right corner, click on the ```Actions``` dropdown menu and select ```Connection details```.
+* Under the ```Public Network``` settings, you will need to take note of the hostname as well as the provided port number.
+* You can secure your database's inbound connections using DigitalOcean's ```Getting Started``` tutorial or by selecting ```Secure this database cluster by restricting access``` under the ```Trusted Sources``` section on the Overview page. You will want to complete this step after you've successfully deployed the SCIM bridge in Step Two (below), so that you can add the ip address of the SCIM bridge's container to that section. 
+
+
+### Step Two: Building and Deploying using App Platform
+
+Now that a redis Droplet has been created, you can start the deployment process of the SCIM bridge. Be sure that you have forked this repo before continuing:
+
+
+#### Setting up the forked repo:
+
+
+* Under ```Manage``` in the left-hand navigation menu, select ```Apps``` or select the ```Create``` dropdown menu in the top right corner of your DigitalOcean tenant and select Apps.
+* Select ```Launch Your App``` on the splash page. If you've already started using Apps, select ```Create App``` in the top right corner of the page.
+* Choose Github as your source. (You may be prompted to walk through an authorization process for your Github account and your DigitalOcean tenant)
+* Choose the repository that contains the files for the DigitalOcean App Platform deployment.
+* Choose the ```main``` branch.
+* You can choose to allow or deny Autodeploy code changes.
+* Click ```Next```.
+
+***NOTE**: DigitalOcean will notify you that it cannot find an app in the repo. This is due to the fact that App Platform expects the Dockerfile to be located at the root of the repo. In this case, the Dockerfile is located in the ```digitalocean-app-platform``` directory and we need to specify that source directory in App Platform.*
```

Comment:
Looks like there's an additional `*` under `NOTE` which is causing this entire block to get bolded.